packages/unit-testing/stubs/workbook/PivotResultsIterator.js

```diff
@@ -0,0 +1,12 @@
+define([], function () {
```

Comment:
This module is not used anywhere. It is probably the return type of the loadPivotResults function of the N/workbook but it's not documented. Let's keep it and improve later if needed.